packages/manager/apps/hpc-vmware-managed-vcd/src/components/datagrid/container/DatagridContainer.component.tsx

```diff
@@ -44,7 +50,15 @@ export default function DatagridContainer({
     setSorting,
   } = useResourcesIcebergV2({
     route: api,
-    queryKey: [icebergListingQueryKey, containerId],
+    queryKey: listingQueryKey,
+  });
+
+  useAutoRefetch({
+    queryKey: listingQueryKey,
+    enabled: hasResourceUpdatingTargetSpec(
+      (flattenData as unknown) as UpdatableResource[],
+    ),
+    interval: 4000,
```

Comment:
You reimplemented `refetchInterval` basically
I think it would be simpler to just add `refetchInterval` prop in `useIcebergV2` hook